dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowFactoryImpl.java

```diff
@@ -228,6 +230,22 @@ private WorkflowStep convertStep(Map<String, Object> row) throws IllegalAccessEx
 		return step;
 	}
 
+	private WorkflowTask convertTask(Map<String, Object> row)

```

Comment:
Do not have to work on it right now, but this should be a DbTransformer in the future